src/bioregistry/data/bioregistry.json

```diff
@@ -100737,6 +100737,33 @@
     ],
     "uri_format": "http://purl.obolibrary.org/obo/SBO_$1"
   },
+  "scancer": {
+    "contact": {
+      "email": "zty2009@hit.edu.cn",
+      "name": "Tianyi Zhao",
+      "orcid": "0000-0001-7352-2959"
+    },
+    "contributor": {
+      "email": "m.naguthana@hotmail.com",
+      "github": "nagutm",
+      "name": "Mufaddal Naguthanawala",
+      "orcid": "0009-0009-5240-7463"
+    },
+    "description": "SCancerRNA database contains experimentally supported data on long ncRNA, microRNA, PIWI-interacting RNA, small nucleolar RNA, and circular RNA biomarkers, as well as data on their differential expression at the cellular level.",
+    "example": "9530",
+    "homepage": "http://www.scancerrna.com/BioMarker/",
+    "name": "SCancerRNA",
+    "pattern": "^\\d{1,4}$",
```

Comment:
Not sure if we should set 4 as an upper limit here. They don't use 0 padding with fixed length for IDs and they increase incrementally so they could eventually go beyond 4 digits. `^\\d+$` might be more robust here.

Reply:
Agreed, updated regex.